cardinal/main.go

```diff
@@ -18,6 +18,14 @@ func main() {
 		log.Fatal().Err(err).Msg("")
 	}
 
+	MustInitWorld(w)
+
+	Must(w.StartGame())
+}
+
+// MustInitWorld registers all components, messages, queries, and systems. This initialization happens in a helper
+// function so that this can be used directly in tests.
+func MustInitWorld(w *cardinal.World) {
```

Comment:
I don't think we want this, seems like an unnecessary hop

Reply:
It's to ensure the test setup code matches the code that will eventually be deployed with `world cardinal start`. If we leave everything in the main function, I don't know how we would inject the test fixture world.